lumicks/pylake/population/dwelltime.py

```diff
@@ -669,12 +694,27 @@ def pdf(self, x):
         x : np.array
             array of independent variable values at which to calculate the PDF.
         """
-        return np.exp(
-            _exponential_mixture_log_likelihood_components(
-                self.amplitudes, self.lifetimes, x, *self._observation_limits
-            )
+        limits = np.vstack(list(np.broadcast(*self._observation_limits)))
+        unique_limits, counts = np.unique(limits, axis=0, return_counts=True)
+
+        pdfs = np.asarray(
+            [
+                (count / sum(counts))
+                * np.logical_and(x >= limits[0], x < limits[1])
```

Comment:
do we still need this bit since the components will be `0` outside the bounds when returned from `_exponential_mixture_log_likelihood_components`?